src/styles/components/location-card.css

```diff
@@ -0,0 +1,84 @@
+.card {
```

Comment:
@esitarz I named this file a little more explicitly and moved the popup styles to a different file. I think the changes are divergent enough now that there probably won't be much opportunity to merge them for now